packages/web3-validator/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "web3-validator",
+	"version": "0.1.0-alpha.0",
```

Comment:
```suggestion
	"version": "1.0.0-alpha.0",
```

Reply:
Its recommended to start with `0.1.0` in [semantic versioning scheme](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).

Reply:
This is okay for now, however we should probably shift back to using Lerna in fixed mode as mentioned [here](https://github.com/ChainSafe/web3.js/discussions/4464), so that all `4.x` packages have the same `4.x` version